src/helpers/management_utils.ts

```diff
@@ -160,6 +160,16 @@ export function reloadAliases() {
     }
 }
 
+export function reloadEndGameMessages() {
+    try {
```

Comment:
Store the current end game messages in a temporary variable, then do the try, and if it fails revert back